tests/IntegrationTests/WorkflowExecutor.IntegrationTests/StepDefinitions/WorkflowTaskArtefactStepDefinitions.cs

```diff
@@ -56,7 +56,7 @@ public async Task GivenIHaveABucketInMinIO(string name)
         public async Task GivenIHaveAPayloadInTheBucket(string folderName, string bucketName, string payloadId)
         {
             _outputHelper.WriteLine("Retrieving pathname");
-            var pathname = Path.Combine(GetDirectory(), "DICOMs", folderName, "dcm");
+            var pathname = Path.Combine(GetDirectory() ?? "", "DICOMs", folderName, "dcm");

```

Comment:
"" in this file should probably be string.Empty;

Reply:
Files been removed